starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,41 @@
+/* ---- reset ---- */
+
+html {
+    box-sizing: border-box;
+}
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
```

Comment:
Use the boilerplate in the cheatsheet. You should have the body reset now:
```
body {
  margin: 0;
  padding: 0;
}
```
And then it should go typography style and so on...
I provide you again the cheatsheet link: https://github.com/ironhack/bcn-webdev-cheatsheet